Explorer/Assets/Scripts/CrdtEcsBridge/Serialization/ProtobufSerializer.cs

```diff
@@ -9,5 +9,19 @@ public void DeserializeInto(T instance, in ReadOnlySpan<byte> data)
         {
             instance.MergeFrom(data);
         }
+
+        public ReadOnlyMemory<byte> Serialize(T model)
+        {
+            //Is it the same as doing model.ToByteArray()?
+            /*Span<byte> buffer = new byte[model.CalculateSize()].AsSpan();
+            model.WriteTo(buffer);
+            return buffer;*/
+
+            var buffer = new byte[model.CalculateSize()];
```

Comment:
Can you think about possible pooling of Serialized byte array? It looks like the lifecycle of these components is fully under our control so ultimately we should reuse arrays.